.github/workflows/release.yml

```diff
@@ -4,21 +4,24 @@ on:
   push:
     tags:
       - "v*.*.*"
+  pull_request:
 
 jobs:
+  build_all:
+    uses: ./.github/workflows/build.yaml
   release:
     runs-on: ubuntu-latest
+    needs: build_all
     steps:
       - uses: actions/checkout@v3
       - run: |
-          bazel build //:for_all_platforms
```

Comment:
got it thanks - you're right I got the notification on a draft PR and this bit just wasn't there yet.